### ISSUE TITLE
Getting spammed in console after using expo-codemod

### ISSUE THREAD

**Issue by spyshower** [expo/expo#4501]
This is what floods my console:

```
1. Add correct versions of these packages to your project using:

   expo install expo-constants

   If "install" is not recognized as an expo command, update your expo-cli installation.

2. Change your imports so they use specific packages instead of the "expo" package:

 - import { Constants } from 'expo' -> import Constants from 'expo-constants'


The following APIs have moved to separate packages and importing them from the "expo" package is deprecated: Constants.
```

I have even changed where I get `Constants` from. Did you accidentally write `while (true)` ?

**Comment by brentvatne** [comment]
ah you caught us, lol - i put`if (user === 'spyshower') { while(true) { spam() }}`

jokes aside, what is most likely happening is there is something your `node_modules` that is using `Constants`. you can search for `Constants` in `node_modules` to see where it's coming from and please let us know. if it's from a package we control we can resolve it, if not you may need to send a pull request to someone or use [patch-package](https://www.npmjs.com/package/patch-package) for now.

**Comment by spyshower** [comment]
> ah you caught us, lol - i put`if (user === 'spyshower') { while(true) { spam() }}`

Haha, yea I am expecting something like that to be honest :)

The 'Constants' search gives so many results that I can't manually check each one individually, so I have to live with this for now.

**Comment by BrentGrammer** [comment]
Getting the same persistent warning about the Constants import.  I am not importing Constants from Expo anywhere in my source code.  Doing a search for Constants in the node_modules folder yields many results as spyshower mentioned and I'm not sure where to start looking or what I'm looking for exactly.

**Comment by victor9000** [comment]
The spam pain is real guys, my terminal logging is pretty much useless due to this issue.  The spam only appears when I run my app on an Android device.  iPhone and emulators don't seem to be affected.  I've narrowed it down to the use of createDrawerNavigator in the react-navigation package.  The DrawerNavigator seems to use something called SafeAreaView which is causing the spam.

@brentvatne You can find a repro in this snack: https://snack.expo.io/@victor9000/cli-spam-demo, but you'll need to export it and run it on an Android 9 device to see the problem.

In the mean time, is there a way to disable these deprecation warnings?

**Comment by martin-ecatch** [comment]
Hi guys, one of these is coming from sentry-expo@1.12.0.

**Comment by raajnadar** [comment]
The title is misleading, I didn't use codemod but still I am getting this warning. I am using react-native-paper and react-navigation I went through rn paper but found no imports from expo. Now it is time to check the react-navigation repo

**Comment by raajnadar** [comment]
Found the issue here https://github.com/react-native-community/react-native-safe-area-view/blob/bf7405b8fb3a42df204255e1fbc05f9cc9c83628/index.js#L91 cc @brentvatne

**Comment by antgel** [comment]
@martin-ecatch Correct, there are currently two open PRs on this for sentry-expo. :laughing:

**Comment by brentvatne** [comment]
hello! please install `expo@33.0.4` and this will likely resolve the issue with the warnings that are coming from libraries like react-navigation, react-native-paper, and so on. related pr: https://github.com/expo/expo/pull/4470

i also published `sentry-expo@1.13.0` to import from `expo-constants`
